projects/commudle-admin/src/app/app.component.scss

```diff
@@ -32,9 +32,13 @@
           margin-left: $space-16;
 
           .menu-user-avatar {
-            width: 45px;
-            height: 45px;
-          }
+              max-width: 40px;
+              max-height: 40px;
+              img{
+                  width: 100%;
+                  border-radius: 50%;
+                }
```

Comment:
Please give an overflow: hidden to the .menu-user-avatar because if the width is 40px and height is more than 40px then that content might overflow and also move the border radius also to the .menu-user-avatar (for the same above reason)

Reply:
Okay